corehq/apps/hqwebapp/widgets.py

```diff
@@ -170,6 +170,6 @@ def render(self, name, value, attrs=None, renderer=None):
         if isinstance(value, dict):
             value = json.dumps(value)
         output = super(GeoCoderInput, self).render(name, value, attrs, renderer)
-        return mark_safe("""
-            <div class="geocoder-proximity">{}</div>
-        """.format(output))
+        return format_html(
+            '<div class="geocoder-proximity">{}</div>',
+            mark_safe(output))  # nosec: must support the input tag
```

Comment:
Could output return a safe string instead, or is that out of scope. If we mark it safe here, it's hard to tell whether we're right or not that it's safe because it's so far from the code that generated it.